### PR TITLE
[nexus] Request larger extents

### PR DESCRIPTION
**Pull request by smklein**

### CODE REVIEW

File: nexus/src/external_api/params.rs [hunk 1]
```diff
@@ -367,7 +367,7 @@ pub struct DiskCreate {
     pub size: ByteCount,
 }
 
-const EXTENT_SIZE: u32 = 1_u32 << 20;
+const EXTENT_SIZE: u32 = 256_u32 << 20;
```
Comment (leftwo) [inline nexus/src/external_api/params.rs:370]: Why not << 1 by some amount instead of << 256?

Comment (smklein) [reply nexus/src/external_api/params.rs:370]: ah, to me, (1 << 20) means "one MiB", so 256 << 20 means "256 MiB".

Is there a different short-hand you'd prefer?

Comment (leftwo) [reply nexus/src/external_api/params.rs:370]: I was not sure if the 256 was special in this case, so I was confused.
That, and that the previous extent_size was 256 (1MiB/4096).
It's probably not worth nit picking on this detail, as we may be tuning it more later, and
we will probably change this again.  The important thing is to increase it.

Also, a 256MiB extent file might be "too big".  I think a next default size we try
should be 64Mib for the file, which would mean extent_size that crucible 
receives is 16384 (assuming 4k block size).

Comment (smklein) [reply nexus/src/external_api/params.rs]: Updated to 64 MiB, added a comment

**Review by leftwo — APPROVED** [review]

**Pull request merged**